rotkehlchen/tests/unit/decoders/test_llamazip.py

```diff
@@ -0,0 +1,418 @@
+    tx_hash = deserialize_evm_tx_hash('0xd75a0b795e0748dffdedc80b731710e9596f6af5fbc77274482cc785ed4c1cd3')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp, fee_amount, spend_amount, receive_amount = TimestampMS(1679521589000), '0.0000419029', '0.125', '215.453696'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(fee_amount)),
+            location_label=arbitrum_one_accounts[0],
+            notes=f'Burned {fee_amount} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.TRADE,
+            event_subtype=HistoryEventSubType.SPEND,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(spend_amount)),
+            location_label=arbitrum_one_accounts[0],
+            notes=f'Swap {spend_amount} ETH in LlamaZip',
+            counterparty=CPT_LLAMAZIP,
+            address=ARBITRUM_ROUTER_ADDRESSES[0],
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.TRADE,
+            event_subtype=HistoryEventSubType.RECEIVE,
+            asset=Asset('eip155:42161/erc20:0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9'),
+            balance=Balance(amount=FVal(receive_amount)),
+            location_label=arbitrum_one_accounts[0],
+            notes=f'Receive {receive_amount} USDT as the result of a swap in LlamaZip',
+            counterparty=CPT_LLAMAZIP,
+            address=ARBITRUM_ROUTER_ADDRESSES[0],
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x9fAe1b2Be0A8D7e64780fd740F8AD05188E8170A']])
+def test_llamazip_arbitrum_swap_token_to_token(database, arbitrum_one_inquirer, arbitrum_one_accounts):  # noqa: E501
+    tx_hash = deserialize_evm_tx_hash('0xb1f65779afe9b92edd791af3b95ea36c46d6a6e8f8cc60f4740bb11b79993a92')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp, fee_amount, spend_amount, receive_amount = TimestampMS(1679511150000), '0.0000360038', '0.071985413648931875', '0.00454144'  # noqa: E501
+    a_weth = Asset('eip155:42161/erc20:0x82aF49447D8a07e3bd95BD0d56f35241523fBab1')
```

Comment:
nitpick: combine those in one line as definitions of the test
```suggestion
    timestamp, fee_amount, spend_amount, receive_amount = TimestampMS(1679511150000), '0.0000360038', '0.071985413648931875', '0.00454144'  # noqa: E501
    a_weth = Asset('eip155:42161/erc20:0x82aF49447D8a07e3bd95BD0d56f35241523fBab1')
```